exp/tests/test_response_views.py

```diff
@@ -1030,7 +1030,7 @@ def test_get_appropriate_children_in_child_csv_as_researcher(self):
             content,
             "Data from unconsented response included in child file download!",
         )
-        self.assertIn(
+        self.assertNotIn(
```

Comment:
Thanks for fixing this!
Would it be better to just delete this whole assertion?

Reply:
Yep, I think that makes sense!